lib/email_tracking_retriever.py

```diff
@@ -170,6 +174,9 @@ def get_items_from_email(self, email_str) -> Any:
   def get_delivery_date_from_email(self, email_str) -> Any:
     pass
 
+  def log_in_if_necessary(self):
```

Comment:
can we add a comment here explaining that by default it does nothing but it can be used for, e.g. amazon login?

Reply:
added